R/util.R

```diff
@@ -178,10 +178,36 @@ list_n <- function(column){
 #' extract elements from each row of list type column or data frame type column
 #' @export
 list_extract <- function(column, position = 1, rownum = 1){
+
+  if(position==0){
+    stop("position 0 is not supported")
+  }
+
   if(is.data.frame(column[[1]])){
-   sapply(column, function(column) column[rownum, position])
+    sapply(column, function(column){
+      if(position<0 & position >= -ncol(column)){
```

Comment:
Can you put if statement outside of sapply so that it does not have to get repeated?